com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandardProgram.hlsl

```diff
@@ -57,6 +57,7 @@
 #pragma shader_feature_local _ _IRIDESCENCE _GRADIENT_FOUR_POINT _GRADIENT_LINEAR
 #pragma shader_feature_local _ENVIRONMENT_COLORING
 #pragma shader_feature_local _ _BLUR_TEXTURE _BLUR_TEXTURE_2 _BLUR_TEXTURE_PREBAKED_BACKGROUND
+#pragma multi_compile_fragment _ _SCREEN_SPACE_OCCLUSION
```

Comment:
I'm guessing this has to be multi_compile vs a shader_feature?

Reply:
I tested and yes, it has to be multi_compile. In our app, user can disable or enable SSAO in settings. With shader_feature, the variant is missing in build and SSAO doesn't work.

If shader build time is really an issue, maybe we can have a custom define like this:
```hlsl
[Toggle(USE_SCREEN_SPACE_OCCLUSION)] _UseScreenSpaceOcclusion("Enable SSAO support", Float) = 0.0

...

#if defined(USE_SCREEN_SPACE_OCCLUSION)
#pragma multi_compile_fragment _ _SCREEN_SPACE_OCCLUSION
#endif
```

But it will require devs to explicitly enable SSAO support on each materials 🤔

Reply:
Ah okay that makes sense. Your toggle idea is a good one, but I agree a little clunky for the end user to have to check that on each material. How about this instead:

Move `#pragma multi_compile_fragment _ _SCREEN_SPACE_OCCLUSION` in GraphicsToolsStandardProgram.hlsl to [here](https://github.com/microsoft/MixedReality-GraphicsTools-Unity/blob/d18aadb89223ac3614906338ec0169f79606857d/com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandard.shader#L185) in GraphicsToolsStandard.shader?

That way we only pay for the extra variants in the non-canvas version of the shader and for URP only? (I'm assuming people don't want SSAO on their canvas UI? 😅)

Reply:
I rebased my branch on `main` and included the change you proposed :)